eq-author-api/app.js

```diff
@@ -45,14 +49,17 @@ app.use(
   pino,
   cors(),
   createAuthMiddleware(logger),
-  loadQuestionnaire,
-  runQuestionnaireMigrations(logger)(require("./migrations"))
+  runQuestionnaireMigrations(logger)(require("./migrations")),
+  userSetup,
```

Comment:
Should this also be set for launch? We could do as part of the permissions changes.